main_window/rocket_data.py

```diff
@@ -222,35 +238,34 @@ def save(self, csv_path):
             # all appearing keys
             keys: List[DataEntryKey] = list(map(
                 lambda data_entry_key: DataEntryKey(data_entry_key.full_address, data_entry_key.data_id),
-                self.existing_entry_keys))
+                set(self.keyset.keys())))
```

Comment:
```suggestion
                self.keyset.keys()))
```
Might not need to be here as well

Reply:
I was not able to get .keys() to work without fully converting to set due to its type (set-like).